heudiconv/convert.py

```diff
@@ -407,6 +409,43 @@ def convert_dicom(item_dicoms, bids_options, prefix,
                 shutil.copyfile(filename, outfile)
 
 
+def convert_physio(item_dicoms, bids_options, prefix):
+    """Save DICOM physiology as BIDS physio files
+    Parameters
+    ----------
+    item_dicoms : list of filenames
+        DICOMs to save
+    bids_options : list or None
+        If not None then save to BIDS format. List may be empty
+        or contain bids specific options
+    prefix : string
+        Conversion outname
+    Returns
+    -------
+    None
+    """
+    if bids_options is not None:
+        try:
```

Comment:
to reduce indentation when possible
```suggestion
    if bids_options is None:
        return
```

Reply:
Thanks for the suggestion!
Fixed in cbinyu/heudiconv@532c630c4170d4db031e78d8aecde35156eb714b
(I didn't accept your suggested change because I wanted to change the indentation of the function in the same commit).